OpenInSublimeText/OSTOpenInSublimeText.m

```diff
@@ -112,7 +112,7 @@ - (void) openCurrentFileWithSublimeTextAtLocation:(CursorLocation)location {
         
         NSTask *task = [[NSTask alloc] init];
         
-        task.launchPath = @"/Applications/Sublime Text.app/Contents/SharedSupport/bin/subl";
+        task.launchPath = @"/Applications/Sublime Text 2.app/Contents/MacOS/Sublime Text 2";
```

Comment:
Ah, yeah. You're right, @pasta12. I didn't realize it was still in beta as I've been using it for quite a long time with no issues. I created #6 for ST2 support.